extensions/amp-access/0.1/amp-access-server.js

```diff
@@ -180,6 +193,39 @@ export class AccessServerAdapter {
     return this.clientAdapter_.pingback();
   }
 
+  /** @override */
+  encodeReaderId(readerId) {
+    return this.getSubtleWithKey_().then(key => {
+      return this.subtle_.encrypt(key, readerId).then(s => {
```

Comment:
I thought I would be able to provide a util method in `crypto-impl.js` for you to use directly before this PR.
But, seems you move master :-)

Nevertheless, could you please extract the code to `crypto-impl.js` for reuse?

```
function encrypt(key, text2encrypt) {
  const keyObject = subtle.importKey(key, ...);
  return subtle.encrypt(keyObject, text2encrypt, ...)
}
```